lib/facade/MonitoringAspect.ts

```diff
@@ -312,8 +313,8 @@ export class MonitoringAspect implements IAspect {
     }
   }
 
-  private monitorRds(node: IConstruct) {
-    const [isEnabled, props] = this.getMonitoringDetails(this.props.rds);
+  private monitorRdsCluster(node: IConstruct) {
+    const [isEnabled, props] = this.getMonitoringDetails(this.props.rdsCluster);
```

Comment:
Since the `rds` prop is only deprecated and not removed yet, we need to actually fall back to it to retain backwards compatibility with something like:

```
this.props.rdsCluster ?? this.props.rds
```